plugins/modules/cloud/misc/proxmox.py

```diff
@@ -608,6 +609,16 @@ def main():
     timeout = module.params['timeout']
     clone = module.params['clone']
 
+    if module.params['unprivileged'] is None:
+        module.params['unprivileged'] = False
+        module.deprecate(
+            'The default value {0} is being deprecated and it will be replaced by {1}'.format(
```

Comment:
It's probably a good idea to let the user know which one we are talking about:  ;-)
```suggestion
            'The default value {0} for the parameter "unprivileged" is being deprecated and it will be replaced by {1}'.format(
```
I will update my documentation for that as well :-)